Apps/Src/ReadSwitches.c

```diff
@@ -43,10 +43,12 @@ void Task_ReadSwitches(void* p_arg) {
         } else {
             Contactors_Disable(ARRAY_CONTACTOR);
             Contactors_Disable(ARRAY_PRECHARGE);
+            Lights_Set(A_CNCTR,OFF);
         }
         
         // motor on/off
         Contactors_Set(MOTOR_CONTACTOR, Switches_Read(IGN_2));
```

Comment:
Should we maybe check if the state of any of the IGN switches changed? This works but it's always being called